src/components/HeaderBar/index.tsx

```diff
@@ -394,9 +463,12 @@ const HeaderBar: NextPage = () => {
   return (
     <div className={i18n.language}>
       <header
-        className={`wrapper header-bar ${
+        className={`header-bar ${headerbarExtraClass} 
+        ${
           !isChrome ? 'opacity' : ''
-        } ${headerbarExtraClass} ${borderBottomVisible ? 'has-bg' : 'has-bg'}`}
+        }
+        ${pathname === '/' && !borderBottomVisible ? 'no-bg' : 'has-bg'}
```

Comment:
if `pathname === '/' && !borderBottomVisible` needs to be used multiple times, please create a variable for it, and reference the variable to make the code better structured.